generator/source/release/chime/notes.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+hide: true
+title: Hyperledger Sawtooth 1.2 (Chime)
+permalink: /release/chime/
+release: 1.2
+release-name: chime
+# Copyright (c) 2019 Cargill Incorporated
+# Licensed under Creative Commons Attribution 4.0 International License
+# https://creativecommons.org/licenses/by/4.0/
+---
+
+<a href="top"></a>
+
+Hyperledger Sawtooth 1.2 (Chime) is now available.  See the [latest
+documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.2.3/) to
+get started.
+
+This page describes the new and changed features in Hyperledger Sawtooth 1.2
+(since release 1.1).  Information is organized in the following categories:
+
+- [New Features](#new-features)
+- [Non-Breaking Changes](#non-breaking-changes)
+- [Documentation Changes](#documentation-changes)
+
+# Hyperledger Sawtooth 1.2
+
+## New Features
+
+- Support for [Sawtooth PBFT consensus engine version
+  1.0](https://sawtooth.hyperledger.org/docs/pbft/releases/1.0.1/).
+- Sawtooth PBFT is the recommended consensus mechanism. It is designed to handle
+  small to medium network sizes with finality and byzantine fault tolerance.
+- Sawtooth PoET is designed for larger networks at the expense of slower /
+  probabilistic finality.
+- Sawtooth Raft is designed for small to medium networks but performs more
+  slowly than Sawtooth PBFT and lacks byzantine fault tolerance.
+- Mobile SDKs for iOS (Swift) and Android (as part of the Java SDK).
+  - [Sawtooth SDK Swift](https://github.com/hyperledger/sawtooth-sdk-swift)
+  - [Sawtooth SDK Java](https://github.com/hyperledger/sawtooth-sdk-java)
+- Support for raw transaction headers, as specified in [Sawtooth RFC
+  #23](https://github.com/hyperledger/sawtooth-rfcs/blob/master/text/0023-raw-txn-header.md).
+  This feature is backward compatible via the use of a protocol version indicator.
+- All core transaction families are compatible with [Sawtooth Sabre release
+  0.4](https://sawtooth.hyperledger.org/docs/sabre/releases/0.4.0/).
+
+[> back to top <](#top)
+
+## Non-breaking changes
+
+- PoET has been patched to
+  - resolve a security defect where a correctly admitted validator could game
+    its wait times. Thanks to Huibo Wang, Guoxing Chen, Yinqian Zhang,
+    Zhiqiang Lin for analyzing the poet implementation and identifying the defect.
+  - support the updated attestation service API
```

Comment:
Please fix the wording and capitalization thusly:

> - PoET has the following patches:
>    - Resolve a security defect ...
>       ...
>    - Support the updated ...

Reply:
Fixed